src/nyx/image_loader1x.cpp

```diff
@@ -40,7 +41,10 @@ bool ImageLoader1x::open(const std::string& fpath)
 		else 
 		{
 			if (Nyxus::check_tile_status(fpath))
-				FL = std::make_unique<NyxusGrayscaleTiffTileLoader<uint32_t>> (n_threads, fpath); 
+				FL = std::make_unique<NyxusGrayscaleTiffTileLoader<uint32_t>> (n_threads, fpath, 

```

Comment:
Why are we passing in globals this way instead of modifying the ImageLoader1x constructor?